### ISSUE TITLE
Microsoft.Rtc.Internal.Media assembly error

### ISSUE THREAD

**Issue by prabhakarreddy1234** [Microsoft/BotBuilder-RealTimeMediaCalling#24]
I am getting this weird error when i try to run the application.  

![image](https://user-images.githubusercontent.com/1984707/28661789-c49ea596-72af-11e7-9387-2d41d6d963fe.png)

I am using Windows 10, VS 2017,  Microsoft.Skype.Bots.Media-1.5.0.1177-alpha, .Net 4.6.

**Comment by ssulzer** [comment]
The process hosting a real-time media bot built with the Microsoft.Skype.Bots.Media library must run in 64-bit mode. Furthermore, only the Azure "classic" Cloud Service and Azure Virtual Machine deployment models are supported. For Azure Cloud Service, we recommend the bot runs in an ASP.NET WebAPI-based Worker role.

**Comment by prabhakarreddy1234** [comment]
I am running it locally with Visual Studio and compiling with`x64` processor.  Still the same issue.

**Comment by ssulzer** [comment]
No, we currently don't have a way to run real-time media bots locally on a developer's machine for testing purposes. The real-time media bot must be deployed in Azure.

**Comment by omerzk** [comment]
Clickbait: Want a quick solution?!
https://stackoverflow.com/a/49425225/3263555

**Comment by voicupul** [comment]
Did anything changed since last year?
Is it possible to run real time media bots locally?

**Comment by vinaychandra** [comment]
@voicupul, please take a look [here](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/articles/Testing.html)
